datafusion/functions-aggregate/src/approx_percentile_cont.rs

```diff
@@ -293,7 +293,7 @@ fn get_approx_percentile_cont_doc() -> &'static Documentation {
 | 65.0                                            |
 +-------------------------------------------------+
 ```"#)
-            .with_argument("expression", "Expression to operate on. Can be a constant, column, or function, and any combination of arithmetic operators.")
+            .with_standard_argument("expression", "The")
```

Comment:
curious, what does `"The"` stand for?

Reply:
https://github.com/apache/datafusion/blob/a8d3fae21d12a9caafad77a11b8a22bcfe9555fd/datafusion/expr/src/udf_docs.rs#L148

Reply:
Thanks for the link. `"The"` still looks weird to me. I guess it gets concatenated with something else.

if i follow the link correctly, this gets passed as `expression_type` argument, but "The" does not look like expression type.

Could we have an overload so that `"The"` is not needed?
eg `with_standard_sole_argument` could produce "The expression to operate on. Can be a constant, column, or function, and any combination of operators." without requiring "The" from the caller.

Reply:
I agree, it really does look strange. .`with_standard_argument("expression", None)` / `with_standard_argument("expression", Some("String"))` ?